lib/pghero/methods/indexes.rb

```diff
@@ -1,49 +1,233 @@
+                    idx_blks_read
+                  FROM
+                    pg_statio_user_indexes
+                  WHERE
+                    indexrelname = replace('%s','$$ || logicalrelid::text || $$', '$$ || indexname::text || $$') $$ )).result::bigint AS idx_blks_read
+              FROM
+                dist_indexes
+            )
+            SELECT
+              schemaname AS schema,
+              relname AS table,
+              indexrelname AS index,
+              CASE WHEN idx_blks_hit + idx_blks_read = 0 THEN
+                0
+              ELSE
+                ROUND(1.0 * idx_blks_hit / (idx_blks_hit + idx_blks_read), 2)
+              END AS hit_rate
+            FROM (
+              SELECT
+                schemaname,
+                tablename AS relname,
+                p.indexname as indexrelname,
+                sum(idx_blks_hit) AS idx_blks_hit,
+                sum(idx_blks_read) AS idx_blks_read
+              FROM
+                placements_idx_blks_hit_read p
+                JOIN dist_indexes d ON p.indexname = d.indexname
+              GROUP BY
+                3, 2, 1
+              UNION ALL
+              SELECT
+                schemaname,
+                relname,
+                indexrelname,
+                idx_blks_hit,
+                idx_blks_read
+              FROM
+                pg_statio_user_indexes s
+              WHERE
+                NOT EXISTS(SELECT * FROM dist_indexes WHERE indexname = indexrelname AND tablename = relname AND schemaname = s.schemaname)
+            ) all_stats
+            ORDER BY
+              3 DESC, 1
+          SQL
+        end
       end
 
       def index_usage
-        select_all <<-SQL
-          SELECT
-            schemaname AS schema,
-            relname AS table,
-            CASE idx_scan
-              WHEN 0 THEN 'Insufficient data'
-              ELSE (100 * idx_scan / (seq_scan + idx_scan))::text
-            END percent_of_times_index_used,
-            n_live_tup AS estimated_rows
-          FROM
-            pg_stat_user_tables
-          ORDER BY
-            n_live_tup DESC,
-            relname ASC
-         SQL
+        if !citus_enabled?
+          select_all <<-SQL
+            SELECT
+              schemaname AS schema,
+              relname AS table,
+              CASE idx_scan
+                WHEN 0 THEN 'Insufficient data'
+                ELSE (100 * idx_scan / (seq_scan + idx_scan))::text
+              END percent_of_times_index_used,
+              n_live_tup AS estimated_rows
+            FROM
+              pg_stat_user_tables
+            ORDER BY
+              n_live_tup DESC,
+              relname ASC
+           SQL
+        else
+          select_all <<-SQL
+            WITH dist_tables_with_indexes (tablename) AS (
+              SELECT 
+                DISTINCT indrelid::regclass
+              FROM
+                pg_index
+                JOIN pg_dist_partition ON indrelid = logicalrelid
```

Comment:
JOINs should be aligned with FROM. (We should have fixed this in previous PRs too)

Reply:
😢

Reply:
Just open another PR for all indentation issues, we can quickly merge it. No need for extensive reviews.